### ISSUE TITLE
WCAG 2.1 Understanding Conformance page fails 1.3.1, 2.4.4 and 4.1.1

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1141]
There's a series of `li` list items not enclosed in a `ul` and the link label for the 6th list item is missing:
https://www.w3.org/WAI/WCAG21/Understanding/conformance.html#conformance-claims-techniques

```
<section id="conformance-claims-techniques">
               <h3>Techniques for Conformance Claims</h3>
               <section id="conformance-claims-sufficient">
                  <li>   
                     <a class="script" href="https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR38">SCR38: Creating a conforming alternate version for a web page designed with progressive enhancement</a>
                  </li>         
...
                  <li>   
                     <a href="https://www.w3.org/WAI/WCAG21/Techniques/failures/f19" class="failure"></a>                     
                  </li>
...
``` 

This `li` items are exposed as role=region with role=listitem children. This is not read as a list in VoiceOver.

**Edit:** role=listitem should be contained by role=list or role=group https://www.w3.org/TR/wai-aria-1.1/#listitem
